build/Makefile

```diff
@@ -501,7 +505,11 @@ testrun : test
 TEST_STATUS:
 	@echo "========================= Performing Building provider tests"
 
-test : TEST_STATUS $(SCXPAL_INTERMEDIATE_DIR) $(INTERMEDIATE_DIR)/testrunner
+TEST_DATA_PREP:
+	@echo "========================= Performing Test data preparation"
+	cp -R $(TEST_DIR)/omi_preexec/testdata/cnfs $(INTERMEDIATE_DIR)/.
```

Comment:
I think I am missing something here.

There is only ONE vendor file that uses include and includedir. That is debian(ubuntu). No body else uses those directives. 

Rest of the cnf files are handwritten to test particular features. 
I am missing something here. 
Firstly, I do not see why the vendor files are to be treated as some sort of gospel - users of MySQL can and do change these.
Secondly, include and includedir are a relatively rare uses.

Also, please see my other comment about how we already differ from how MySQL treats these values and we do.